docs/source/specs/typespec/main.tsp

```diff
@@ -103,16 +103,26 @@ namespace Workspaces {
         ...Timestamps;
     }
 
+    model WorkspaceAncestry {
+        @key uuid: UUID;
+        parent_id?: UUID;
+        name: string;
+    }
+
     alias CreateWorkspaceRequest = BasicWorkspace;
 
     model CreateWorkspaceResponse extends BasicWorkspace{
```

Comment:
@coderbydesign Maybe we need to change here to `Workspace` instead of `BasicWorkspace` once the JSON response has all the fields.

Reply:
Updated!